generators/generator-microsoft-bot-command-list/generators/app/index.js

```diff
@@ -0,0 +1,33 @@
+'use strict';
+const Generator = require('yeoman-generator');
+
+module.exports = class extends Generator {
+  constructor(args, opts) {
+    super(args, opts);
+
+    this.argument("botName", { type: String, required: true });
+    
+  }
+
+  initializing() {
+    this.composeWith(
+      require.resolve('generator-adaptive-bot/generators/app'),
+      {
+        arguments: this.args,
+        packageReferences: [
+          // PLACE COMPONENT PACKAGES THAT YOUR BOT TEMPLATE USES HERE
```

Comment:
Just leave as empty and remove the comment - don't think this matters so much here since it's not a template.